hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -278,6 +278,210 @@ def test_failure_reason(reason):
     '''
     return reason
 
+def test_mount_attrs(mount_name,attribute):
+    '''
+    Ensure that a given directory is mounted with appropriate attributes
+    '''
+    output = _execute_shell_command('mount | grep ' + mount_name)
+    if output.strip() == '':
+        return 'No match'
+    elif attribute not in output:
+        return output
+    else:
+        return True
+
+def service_not_enabled(service_name):
+    '''
+    Ensure that the given service is not running on the system. Return False if the service is running.
+    Return True otherwise
+    '''
+    output = _execute_shell_command('systemctl is-enabled ' + service_name + ' 2>/dev/null')
+    print output
+    if output.strip() == 'enabled':
```

Comment:
Even if output is 'static' it means it's enabled.
Correct way should be something like 
`systemctl is-enabled foo.service; echo $?`
0 indicates that it is enabled. 1 indicates that it is disabled